WcaOnRails/app/helpers/competitions_helper.rb

```diff
@@ -4,13 +4,19 @@ def competition_message_for_user(competition, user)
     messages = []
     registration = competition.registrations.find_by_user_id(user.id)
     if registration
-      messages << "You are " + (registration.accepted? ? "registered." : "currently on the waiting list.")
+      # i18n-tasks-use t('competitions.messages.registered')
+      # i18n-tasks-use t('competitions.messages.waiting_list')
+      messages << t('competitions.messages.registration_tooltip', message: registration.accepted? ? t('competitions.messages.registered') : t('competitions.messages.waiting_list'))
```

Comment:
Hmm... why the `i18n-tasks-use` comments? Does i18n-tasks not handle multiple translations per line?

Would it simpler to just split this into 2 strings? Right now we have 3, and it may not work for all languages.

- "You are registered"
- "You are currently on the waiting list"